command/oauth/cmd.go

```diff
@@ -282,24 +297,35 @@ func oauthCmd(c *cli.Context) error {
 		return errors.New("flag '--client-id' required with '--provider'")
 	}
 
+	_, isDeviceFlow := os.LookupEnv("DEVICE")
+	if c.Bool("console") && !(opts.Provider == "google" || strings.Contains(opts.Provider, "google.com")) {
```

Comment:
We can be more explicit, the google configuration endpoint will always start with `https://accounts.google.com`, for both companies and regular users.